app/controllers/lists_controller.rb

```diff
@@ -112,10 +112,28 @@ def update
     end
   end
 
+  def edit_list_items
+    @list = @tag.lists.find(params[:id])
+  end
+
```

Comment:
Are we missing some tests here? Were these already covered by some feature specs that still pass now that we've switched an endpoint over?

Reply:
Good shout. I was awaiting of an outcome from a discussion i was going to have with Ollie on whether i should expand the feature specs to cover all the controller behaviour or add controller specs, but i think i'll just add in some controller specs. Thanks!